src/views/order_list_view.js

```diff
@@ -0,0 +1,100 @@
+import Backbone from 'backbone';
+import $ from 'jquery';
+import _ from 'underscore';
+// import QuoteView from '../views/quote_view';
+// import Quote from '../models/quote';
+import OrderView from '../views/order_view';
+import Order from '../models/order';
+import Quote from '../models/quote';
+
+const OrderListView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.ordersTemplate;
+    this.listenTo(this.model, 'update', this.render);
+    this.quotes = params.quotes;
+    this.bus = params.bus;
+  },
+  render() {
+    this.$('#orders').empty();
+    this.model.forEach((order) => {
+      const orderView = new OrderView({
+        model: order,
+        template: this.template,
+        tagName: 'li',
+        className: 'order',
+        bus: this.bus,
+      });
+      // this.listenTo(this.bus, 'quoteViewSendsQuoteChange', this.checkChange);
+      this.$('#orders').append(orderView.render().$el);
+    });
+    return this;
+  },
+  events: {
+    'click .btn-buy': 'buyOrder',
+    'click .btn-sell': 'sellOrder',
+  },
+  buyOrder: function(event) {
+    // console.log('You pressed the Buy Order button.');
+    event.preventDefault();
+    this.addToOrders(event, true);
+  },
+  sellOrder: function(event) {
+    // console.log('You pressed the Sell Order button.');
+    event.preventDefault();
+    this.addToOrders(event, false);
+  },
+  addToOrders: function(event, buyIsTrue) {
+    const orderData ={};
+    const symbol = this.$(`select[name=symbol] option:selected`).val();
+    orderData['symbol'] = symbol;
+    console.log(symbol);
+
+    const price = parseFloat(this.$('input[name=price-target]').val());
+    orderData['targetPrice'] = price;
+    // console.log(price);
+
+    orderData['buy'] = buyIsTrue;
+
+    //finds the current market price and adds it to the orderData hash, to be validated in the model.
+    const currentQuoteModel = this.quotes.find({symbol: symbol});
+    currentQuoteModel.set('buy', buyIsTrue);
```

Comment:
Why add a `buy` attribute to `Quote`?  Instead why not let the `Order` keep track of if it's buying or selling?  Your solution limits you to only having buy orders or sell orders not a mix.